lustre-server-patchless.repo

```diff
@@ -1,6 +1,6 @@
 [lustre]
 name=Lustre Server
-baseurl=https://downloads.whamcloud.com/public/lustre/lustre-2.12.2/el7/patchless-ldiskfs-server/
+baseurl=https://build.whamcloud.com/view/lustre/job/lustre-b2_12/52/arch%3Dx86_64%2Cbuild_type%3Dserver%2Cdistro%3Del7.7%2Cib_stack%3Dinkernel/artifact/artifacts/
```

Comment:
this should be b2_12-patchless, build 46 (Minh ran a el7.7 build for patchless)
https://build.whamcloud.com/view/Non-Reviews/job/lustre-b2_12-patchless/arch=x86_64%2Cbuild_type=server%2Cdistro=el7.7%2Cib_stack=inkernel/46/artifact/artifacts/